### DISCUSSION TITLE
Oktopus Project

### DISCUSSION THREAD

**Discussion by leandrofars** [OktopUSP/oktopus#15] [General]
Here you can post anything you need related to the project.

**Comment by makver9** [comment]
I have cloned the latest code and run docker compose command. The UI comes up fine but what credentials should I use to login ? Plus I can't quite register. Please provide login credentials for a locally hosted server.

**Reply by leandrofars** [reply]
It should open a page for you to register user and password.

**Reply by leandrofars** [reply]
Can you check if all containers are running? Specially the "controller" and "mongo_usp"

**Reply by makver9** [reply]
Yes they do run, but even when I try to register, it gives a JSON error

**Reply by makver9** [reply]
![image](https://github.com/user-attachments/assets/91fb9a3b-6fbe-4922-9be7-72e04ff5d12c)
Like that one

**Reply by leandrofars** [reply]
It seems to be a problem with nginx.
1 - How is your nginx config?
2 - Fetch the latest docker images, restart docker compose and refresh browser. Your UI is atill in the old way.
3 - It would be good to have the web browser console network tab print to help debugging.

**Reply by leandrofars** [reply]
Looks like you have an old codebase, It's best to delete all oktopus containers and images. Then clone the repo again and start a fresh installation: 
https://docs.oktopus.app.br/getting-started/installation

**Comment by makver9** [comment]
Deleted all the existing docker images and repo.

Cloned the repo from the link you shared. 

The UI is updated now but I still can't quite register a new user. 


![image](https://github.com/user-attachments/assets/617bed69-5e20-491e-85d8-f17989d7374b)
![image](https://github.com/user-attachments/assets/5aab9da8-663c-4e20-aab1-6dfdb0b07957)
![image](https://github.com/user-attachments/assets/c55149fa-1b2d-4276-9351-3ad4f4754a3b)

**Comment by makver9** [comment]
Let me know what other details I can share for you to find the problem.

**Comment by makver9** [comment]
The network console output

![image](https://github.com/user-attachments/assets/de56a7c2-9729-4336-accf-37828b4bfbdc)

**Reply by leandrofars** [reply]
Open the localhost:80 not localhost:3000

**Reply by luizmeloDev** [reply]
How to fix mongo_usp failure?

**Reply by leandrofars** [reply]
Mongo has some issues depending on the CPU architecture which makes it crash, so if that's the case with you, change the [docker-compose file](https://github.com/OktopUSP/oktopus/blob/main/deploy/compose/docker-compose.yaml#L41) to a different MongoDB version compatible with the server hardware.

**Comment by makver9** [comment]
![image](https://github.com/user-attachments/assets/de1c895f-7d39-406b-bf8c-e53a1f2ffafa)


localhost:80 works, thank you so much for your help on this.

**Reply by makver9** [reply]
Also, what config is the TR369 agent to set at its end for WS connection to the OktopusController. 

MQTT and STOMP works fine for us, does the WS need any change other than what's mentioned at  oktopus-main/agent/oktopus-stomp-obuspa.txt

I get this error when I use same config in the txt with WS. 

Attempting to connect to host=192.168.29.191 (port=8080, unencrypted, path=/ws/agent/) from interface=nas0_0
HandleWscEvent_Error: Error (HS: ws upgrade response not 101) occurred on websocket connection to endpoint_id=oktopusController (host=)
HandleWscEvent_Destroyed: Closed connection to oktopusController
ScheduleWsclientRetry: Retrying connection in 7 seconds (retry_count=1)

**Reply by nick880606** [reply]
do you fix this issue?

**Comment by mohamedyangui** [comment]
the oktopus/stomp server is it modifiable for the queues ?

**Reply by leandrofars** [reply]
Yes, but you get do it in the oktopus source code.

**Reply by luizmeloDev** [reply]
![Captura de tela 2025-07-03 111816](https://github.com/user-attachments/assets/e62dd037-7f3b-454d-b1c7-fca4ed279c1f)

How configuration a cpe here tr069?

**Reply by leandrofars** [reply]
this link may help: https://docs.oktopus.app.br/device-configuration/simulators/usp-agent-simulator

**Comment by Mafia-Baba26** [comment]
How do you man start that?

**Comment by Mafia-Baba26** [comment]
How do you man start that?
